### ISSUE TITLE
Question regarding reproducing paper results

### ISSUE THREAD

**Issue by Alexmsit** [tudelft-iv/view-of-delft-dataset#54]
Hi everyone,

First of all, thank you for publishing this great dataset. Right now i am working on reproducing the paper results.
I followed the instructions within PP-Radar.md and get nearly the same mAP-scores as reported in the paper.
But the classwise AP-scores seem to differ from the paper-results to some extent.

Here is the comparison of the results for training of PP on single-frame radar data with all 7 features used:

![results](https://github.com/tudelft-iv/view-of-delft-dataset/assets/63818660/29ecff68-201e-4a31-86b5-511895210b74)

The first line shows the paper results and the second line shows the reproduction results.
Now my question is why the classwise AP-scores differ up to ~10% from the paper results.
Did you use any additional configuration like a weighted loss-function or other things which would explain this difference, or am i missing something here?

Thanks in advance!
Best regards,

Alex

**Comment by yangyt37** [comment]
May I ask how you can obtain the results for pedestrians and cyclists? The map I obtained using the official evaluation indicators of openpcdet is less than 1. My result is in the following tabel.

**Comment by andraspalffy** [comment]
@Alexmsit thanks for your interest. 10% difference is unfortunately plausible given the dataset size.
Also I guess you are using the validation set, feel free to send it for a test set eval.

@yangyt37 your results suggest that you use a wrong evaluation approach. See this comment which points out where was this answered previously.

https://github.com/tudelft-iv/view-of-delft-dataset/issues/8#issuecomment-1231418778

**Comment by yangyt37** [comment]
I use right evaluation approach and I got the result like this:
Car: 32.760911122320536 
Pedestrian: 27.134859105528225 
Cyclist: 59.60893396401781 
mAP: 39.83490139728885 
Driving corridor area: 
Car: 68.9206407782023 
Pedestrian: 36.17506242356443 
Cyclist: 80.8428254073373 
mAP: 61.979509536368006

**Comment by min-zhang** [comment]
> I use right evaluation approach and I got the result like this: Car: 32.760911122320536 Pedestrian: 27.134859105528225 Cyclist: 59.60893396401781 mAP: 39.83490139728885 Driving corridor area: Car: 68.9206407782023 Pedestrian: 36.17506242356443 Cyclist: 80.8428254073373 mAP: 61.979509536368006

you can provide me with the dataset, and I can help you with the calculations.Four sets of data in total.
1.
INFO  Car AP@0.70, 0.70, 0.70:
bbox AP:29.2754, 37.7937, 31.9468
bev  AP:38.4106, 46.0381, 39.4115
3d   AP:16.5922, 20.8887, 17.5489
aos  AP:25.90, 36.16, 30.63
Car AP_R40@0.70, 0.70, 0.70:
bbox AP:26.3670, 34.6987, 29.4077
bev  AP:35.5528, 43.5953, 36.7469
3d   AP:11.8746, 16.0044, 12.7312
aos  AP:22.75, 33.00, 27.84
Car AP@0.70, 0.50, 0.50:
bbox AP:29.2754, 37.7937, 31.9468
bev  AP:55.4508, 57.8449, 50.1450
3d   AP:39.2968, 46.3682, 39.5512
aos  AP:25.90, 36.16, 30.63
Car AP_R40@0.70, 0.50, 0.50:
bbox AP:26.3670, 34.6987, 29.4077
bev  AP:54.1486, 57.2201, 48.2378
3d   AP:36.6693, 43.8487, 36.9838
aos  AP:22.75, 33.00, 27.84
Pedestrian AP@0.50, 0.50, 0.50:
bbox AP:28.9486, 27.1331, 26.0896
bev  AP:26.8249, 21.8494, 20.8565
3d   AP:16.8029, 14.0226, 13.2201
aos  AP:28.50, 27.83, 24.77
Pedestrian AP_R40@0.50, 0.50, 0.50:
bbox AP:26.4394, 22.8352, 20.6211
bev  AP:23.0455, 18.6491, 16.5344
3d   AP:11.3486, 9.1428, 7.9505
aos  AP:25.77, 22.05, 19.89
Pedestrian AP@0.50, 0.25, 0.25:
bbox AP:28.6086, 27.1331, 26.0896
bev  AP:48.5453, 41.1766, 39.4566
3d   AP:39.2645, 36.5330, 30.8388
aos  AP:29.50, 26.83, 24.77
Pedestrian AP_R40@0.50, 0.25, 0.25:
bbox AP:26.4394, 22.8352, 20.6211
bev  AP:47.7989, 39.7481, 35.1812
3d   AP:38.2997, 32.1196, 28.0489
aos  AP:25.77, 22.05, 19.89
Cyclist AP@0.50, 0.50, 0.50:
bbox AP:63.7330, 61.2801, 54.3684
bev  AP:71.2002, 63.0600, 57.1477
3d   AP:39.1116, 37.5996, 32.8188
aos  AP:63.59, 61.12, 54.23
Cyclist AP_R40@0.50, 0.50, 0.50:
bbox AP:65.6098, 59.1844, 52.9149
bev  AP:71.2757, 64.1361, 57.6604
3d   AP:36.5646, 33.4715, 29.9994
aos  AP:65.46, 59.03, 52.78
Cyclist AP@0.50, 0.25, 0.25:
bbox AP:63.7330, 61.2801, 54.3684
bev  AP:85.9964, 77.8251, 69.9316
3d   AP:85.4105, 73.1937, 67.3817
aos  AP:63.59, 61.12, 54.23
Cyclist AP_R40@0.50, 0.25, 0.25:
bbox AP:65.6098, 59.1844, 52.9149
bev  AP:86.9980, 79.1704, 72.0699
3d   AP:81.9620, 72.9490, 66.0283
aos  AP:65.46, 59.03, 52.78
2.
 INFO  Car AP@0.70, 0.70, 0.70:
bbox AP:25.2931, 35.8688, 30.2304
bev  AP:34.3534, 41.2079, 37.0070
3d   AP:13.5977, 17.5404, 14.6623
aos  AP:22.00, 34.01, 28.79
Car AP_R40@0.70, 0.70, 0.70:
bbox AP:21.6496, 32.8158, 26.9882
bev  AP:31.7833, 40.4042, 33.8964
3d   AP:7.0489, 12.1813, 9.6848
aos  AP:17.99, 30.88, 25.39
Car AP@0.70, 0.50, 0.50:
bbox AP:25.2931, 35.8688, 30.2304
bev  AP:53.9248, 58.3119, 50.3598
3d   AP:34.7278, 41.2159, 36.9743
aos  AP:22.00, 34.01, 28.79
Car AP_R40@0.70, 0.50, 0.50:
bbox AP:21.6496, 32.8158, 26.9882
bev  AP:52.7421, 57.3406, 49.3869
3d   AP:31.6778, 40.5127, 33.8923
aos  AP:17.99, 30.88, 25.39
Pedestrian AP@0.50, 0.50, 0.50:
bbox AP:28.7664, 26.7176, 22.2800
bev  AP:26.4072, 21.3220, 20.4711
3d   AP:16.2190, 13.5128, 13.1590
aos  AP:28.29, 26.09, 21.88
Pedestrian AP_R40@0.50, 0.50, 0.50:
bbox AP:26.2926, 21.6179, 19.5190
bev  AP:22.2260, 17.9081, 16.0608
3d   AP:9.9817, 8.1055, 6.9494
aos  AP:25.61, 20.92, 18.87
Pedestrian AP@0.50, 0.25, 0.25:
bbox AP:28.7664, 26.7176, 22.2800
bev  AP:48.2595, 40.4788, 33.9854
3d   AP:38.4618, 31.6497, 30.0935
aos  AP:28.29, 26.09, 21.88
Pedestrian AP_R40@0.50, 0.25, 0.25:
bbox AP:26.2926, 21.6179, 19.5190
bev  AP:46.2196, 37.6173, 33.2550
3d   AP:36.3881, 30.1446, 26.2724
aos  AP:25.61, 20.92, 18.87
Cyclist AP@0.50, 0.50, 0.50:
bbox AP:60.9057, 58.7827, 52.4480
bev  AP:70.8855, 63.8257, 56.6782
3d   AP:36.7310, 31.9714, 31.0531
aos  AP:60.82, 58.68, 52.36
Cyclist AP_R40@0.50, 0.50, 0.50:
bbox AP:62.0980, 56.1747, 50.1514
bev  AP:71.2272, 63.1126, 56.4553
3d   AP:33.3966, 29.5327, 26.2988
aos  AP:62.00, 56.07, 50.06
Cyclist AP@0.50, 0.25, 0.25:
bbox AP:60.9057, 58.7827, 52.4480
bev  AP:84.8932, 76.9164, 68.6398
3d   AP:81.7922, 73.7306, 66.0486
aos  AP:60.82, 58.68, 52.36
Cyclist AP_R40@0.50, 0.25, 0.25:
bbox AP:62.0980, 56.1747, 50.1514
bev  AP:86.0499, 78.9152, 71.5783
3d   AP:81.5084, 72.7171, 65.6249
aos  AP:62.00, 56.07, 50.06
3.
 INFO  Car AP@0.70, 0.70, 0.70:
bbox AP:24.8006, 34.9097, 30.2909
bev  AP:27.1302, 40.0670, 36.2988
3d   AP:13.2993, 18.0958, 16.3050
aos  AP:21.40, 33.08, 28.71
Car AP_R40@0.70, 0.70, 0.70:
bbox AP:20.8681, 32.3995, 26.9079
bev  AP:23.1890, 37.6880, 32.3889
3d   AP:7.1297, 12.4595, 10.6641
aos  AP:17.13, 30.26, 25.11
Car AP@0.70, 0.50, 0.50:
bbox AP:24.8006, 34.9097, 30.2909
bev  AP:37.1209, 50.3730, 46.3904
3d   AP:28.6340, 40.8486, 36.4850
aos  AP:21.40, 33.08, 28.71
Car AP_R40@0.70, 0.50, 0.50:
bbox AP:20.8681, 32.3995, 26.9079
bev  AP:34.3201, 50.2684, 43.2921
3d   AP:25.0854, 39.3342, 33.1041
aos  AP:17.13, 30.26, 25.11
Pedestrian AP@0.50, 0.50, 0.50:
bbox AP:24.5727, 20.5609, 19.6205
bev  AP:14.6832, 12.6827, 12.3879
3d   AP:10.4197, 10.2574, 10.2718
aos  AP:19.98, 16.95, 16.21
Pedestrian AP_R40@0.50, 0.50, 0.50:
bbox AP:19.9313, 17.3431, 14.5054
bev  AP:8.1676, 7.2403, 6.1752
3d   AP:3.3720, 2.7861, 2.5045
aos  AP:14.75, 12.32, 10.34
Pedestrian AP@0.50, 0.25, 0.25:
bbox AP:24.5727, 20.5609, 19.6205
bev  AP:26.8644, 25.7079, 21.2757
3d   AP:24.6688, 20.7134, 19.6651
aos  AP:19.98, 16.95, 16.21
Pedestrian AP_R40@0.50, 0.25, 0.25:
bbox AP:19.9313, 17.3431, 14.5054
bev  AP:23.2809, 20.2592, 18.1959
3d   AP:20.0765, 17.3011, 14.4550
aos  AP:14.75, 12.32, 10.34
Cyclist AP@0.50, 0.50, 0.50:
bbox AP:38.5128, 33.7345, 32.5540
bev  AP:26.4603, 23.2087, 21.9962
3d   AP:12.1961, 11.8660, 11.3110
aos  AP:35.88, 31.53, 30.32
Cyclist AP_R40@0.50, 0.50, 0.50:
bbox AP:35.0434, 31.5323, 28.2967
bev  AP:20.7259, 18.4099, 16.7069
3d   AP:10.1833, 9.2838, 8.3496
aos  AP:32.18, 28.87, 25.73
Cyclist AP@0.50, 0.25, 0.25:
bbox AP:38.5128, 33.7345, 32.5540
bev  AP:37.9433, 33.2410, 32.0942
3d   AP:37.3923, 32.7788, 31.2410
aos  AP:35.88, 31.53, 30.32
Cyclist AP_R40@0.50, 0.25, 0.25:
bbox AP:35.0434, 31.5323, 28.2967
bev  AP:35.4319, 31.0019, 27.7176
3d   AP:33.8716, 30.3502, 27.0863
aos  AP:32.18, 28.87, 25.73
4.
INFO  Car AP@0.70, 0.70, 0.70:
bbox AP:20.9696, 34.1555, 28.5980
bev  AP:24.3370, 37.2360, 31.4913
3d   AP:11.8908, 15.6095, 13.9356
aos  AP:20.11, 33.27, 27.87
Car AP_R40@0.70, 0.70, 0.70:
bbox AP:17.4207, 30.2644, 24.8281
bev  AP:20.5017, 34.3673, 29.3359
3d   AP:5.0783, 9.9259, 8.1489
aos  AP:16.48, 29.34, 24.06
Car AP@0.70, 0.50, 0.50:
bbox AP:20.9696, 34.1555, 28.5980
bev  AP:36.3833, 48.9255, 41.4244
3d   AP:25.1129, 37.3397, 31.5593
aos  AP:20.11, 33.27, 27.87
Car AP_R40@0.70, 0.50, 0.50:
bbox AP:17.4207, 30.2644, 24.8281
bev  AP:33.8863, 47.7292, 40.8661
3d   AP:21.0562, 34.5135, 28.5334
aos  AP:16.48, 29.34, 24.06
Pedestrian AP@0.50, 0.50, 0.50:
bbox AP:28.2217, 26.3297, 22.0450
bev  AP:23.2074, 19.6977, 18.6556
3d   AP:8.9402, 8.5254, 8.2754
aos  AP:27.91, 25.91, 21.79
Pedestrian AP_R40@0.50, 0.50, 0.50:
bbox AP:24.7021, 21.3426, 19.1736
bev  AP:18.1590, 15.1557, 13.4487
3d   AP:7.0569, 6.0255, 5.1102
aos  AP:24.31, 20.93, 18.78
Pedestrian AP@0.50, 0.25, 0.25:
bbox AP:28.2217, 26.3297, 22.0450
bev  AP:44.7856, 37.7366, 32.3111
3d   AP:35.4566, 30.0252, 28.1741
aos  AP:27.91, 25.91, 21.79
Pedestrian AP_R40@0.50, 0.25, 0.25:
bbox AP:24.7021, 21.3426, 19.1736
bev  AP:41.4406, 33.7527, 31.0403
3d   AP:32.6915, 27.1192, 23.5528
aos  AP:24.31, 20.93, 18.78

Cyclist AP@0.50, 0.50, 0.50:
bbox AP:56.9337, 50.9681, 44.7280
bev  AP:46.0885, 44.7921, 39.6599
3d   AP:23.9402, 22.0751, 21.4132
aos  AP:56.83, 50.86, 44.63
Cyclist AP_R40@0.50, 0.50, 0.50:
bbox AP:55.3930, 49.5998, 44.1024
bev  AP:43.7378, 40.7672, 35.9730
3d   AP:17.9729, 17.2490, 15.0542
aos  AP:55.30, 49.50, 44.00
Cyclist AP@0.50, 0.25, 0.25:
bbox AP:56.9337, 50.9681, 44.7280
bev  AP:72.5824, 65.1337, 62.6713
3d   AP:67.8893, 61.3175, 54.7505
aos  AP:56.83, 50.86, 44.63
Cyclist AP_R40@0.50, 0.25, 0.25:
bbox AP:55.3930, 49.5998, 44.1024
bev  AP:73.7297, 67.1657, 60.6393
3d   AP:66.5659, 60.2604, 54.0202
aos  AP:55.30, 49.50, 44.00

**Comment by fthbng77** [comment]
I need to access the VoD dataset for my graduation project, but I can't access it without filling out the form. What should I do?
